Dockerfile-0.12.6.1

```diff
@@ -0,0 +1,43 @@
+# first stage: build kwkhtmltopdf_server
+
+FROM --platform=linux/amd64 docker.io/golang:1.11
+WORKDIR /tmp
+COPY server/kwkhtmltopdf_server.go .
+RUN go build kwkhtmltopdf_server.go
+
+# second stage: server with wkhtmltopdf
+
+FROM --platform=linux/amd64 docker.io/ubuntu:20.04
```

Comment:
Could we use 22.04? I think there is jammy version of wkhtmltopdf.

Reply:
I could only find a release for the version 16, 18, and 20 of Ubuntu on the 0.12.6-1 [release page](https://github.com/wkhtmltopdf/packaging/releases/0.12.6-1)

Reply:
And maybe that's why our Odoo.sh instances (or anyone's?) are running on Ubuntu 20.04

Reply:
Done. The generated file is 4 bytes bigger than what is generated by Odoo.sh in my report, but visually identical, I'd say this is good enough, what do you think?

Reply:
Sounds good. Maybe you can even remove the `bullseye` thing, now, since ubuntu 22 has glibc 2.35.

Reply:
You are right, it does work now

Reply:
> And maybe that's why our Odoo.sh instances (or anyone's?) are running on Ubuntu 20.04

FWIW, I just looked at an odoo.sh instance with Odoo 17, Ubuntu 22, wkhtmltopdf 0.12.5.

Reply:
Ok, so it's our old Odoo 14 that are running in Ubuntu 20.04 and wkhtmltopdf 0.12.6